book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative('./rental')
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly convert the `rentals` to `attr_accessor` as you already use it for writing in the Rental class initialize method.

Reply:
Ok, thanks for the clarification.